client/actions/checks.js

```diff
@@ -4,34 +4,35 @@ import { PENDING, SUCCESS, ERROR } from '../actions/status'
 export const PUT_CHECK = Symbol('put check')
 export const DELETE_CHECK = Symbol('delete check')
 
-const putCheck = (status, payload = null) => ({
+const putCheck = (status, check, payload = null) => ({
   type: PUT_CHECK,
   status,
+  check,
```

Comment:
The 'check' should actually be part of the payload as we are following the [flux-standard-action](https://github.com/acdlite/flux-standard-action#actions) pattern.